app/javascript/.storybook/main.js

```diff
@@ -2,6 +2,8 @@ const path = require('path');
 const marked = require('marked');
 const renderer = new marked.Renderer();
 
+const prettierConfig = require('../../../.prettirc');
```

Comment:
refactor (blocking): There's a spelling mistake in the name of the prettier config filename.

```suggestion
const prettierConfig = require('../../../.prettierrc');
```